openslides_backend/action/actions/organization/initial_import.py

```diff
@@ -75,6 +79,27 @@ def check_empty_datastore(self) -> None:
         ):
             raise ActionException("Datastore is not empty.")
 
+    def check_migration_index(self, instance: Dict[str, Any]) -> None:
+        data_migration_index = instance["data"].get("_migration_index")
+        if data_migration_index is None:
+            raise ActionException(
+                "Data must have a valid migration index in `_migration_index`."
+            )
+        if data_migration_index < 1:
+            raise ActionException(
+                f"Data must have a valid migration index >= 1, bus has {data_migration_index}."
+            )
+
+        backend_migration_index = get_backend_migration_index()
+        if backend_migration_index < data_migration_index:
```

Comment:
```suggestion
        if backend_migration_index == data_migration_index:
```
The backend can't function with older data

Reply:
> The backend can't function with older data
I think you mean !=, not ==, to raise the exception.

Reply:
Right, my bad :D

Reply:
After our conversion: I will not change this. If the backend_migration_index is greater than the MI from import-data, there will be a message in the result. In this case the backend will not accept any action, until the needed migrations are done.